format_checker/common_checks.py

```diff
@@ -23,6 +25,18 @@
 }
 
 
+def check_repo_archived(checked_projects, filename, row, i, log):
+    project_url = row["Project URL"]
+    try:
+        resp = requests.get(project_url)
+        # Determine if it is an archived project
+        if "This repository has been archived by the owner. It is now read-only." in resp.text:
+            log_archived_error(filename, log, i, row, "Project URL")
+    except requests.exceptions.RequestException as e:
+        # handle(e)
+        pass
```

Comment:
Instead of simply passing here, it would be good to warn that the check for repo archived failed along with outputting the exception

Reply:
get it, I change ```pass``` to ```log_esp_error(filename, log, "The check for repo-archived failed due to ERROR:" + str(e))```

yes the PR is similar to 334, maybe the two funcitions can merge into one ?